### ISSUE TITLE
getaddrinfo failed

### ISSUE THREAD

**Issue by WhiteSymmetry** [startsts/cryptohubminer#1]
Traceback (most recent call last):
  File "miner.py", line 49, in <module>
  File "urllib2.pyc", line 154, in urlopen
  File "urllib2.pyc", line 429, in open
  File "urllib2.pyc", line 447, in _open
  File "urllib2.pyc", line 407, in _call_chain
  File "urllib2.pyc", line 1241, in https_open
  File "urllib2.pyc", line 1198, in do_open
urllib2.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

**Comment by startsts** [comment]
yes I know! Windows 10 blocks my miner! I will appreciate if you investigate the problem. I can pay you some reward in CROC coin.  I am not windows user, I can do nothing

**Comment by ItIsCRYT** [comment]
What am I so unlucky, just about 6 days ago I tried to run on Linux, it did not go. Today I'm trying on Windows 10, I also do not go ... I reset it 4 times, well at least this post found

**Comment by startsts** [comment]
@ItIsCRYT better to say I'm a bad programmer that can't make stable version for windows nor linux

**Comment by ItIsCRYT** [comment]
@startsts Notice that I did not say that. I understand why there is no stable version on Linux, because people are afraid of Linux and think that there is something complicated

**Comment by startsts** [comment]
you are right. But I just dont have time to make stable versions for them all. I spend on time on my pool/exchange
